sql/src/main/java/io/crate/metadata/Reference.java

```diff
@@ -65,28 +68,32 @@ public Reference(StreamInput in) throws IOException {
         columnStoreDisabled = in.readBoolean();
     }
 
-    public Reference(ReferenceIdent ident,
+    public Reference(@Nullable Integer position,
```

Comment:
personal preference: I'd have add it as the last arg because it is 1) nullable, 2) meta information